docs/FAQ/FAQ-General.md

```diff
@@ -168,6 +169,13 @@ BTCPay generates a different address for each invoice. The address is derived fr
 
 At this time, it is not possible to instantly convert your payments into fiat through BTCPay. This feature is planned for the future, but there is no ETA.
 
+## What if I have a problem with a paid invoice?
+
+BTCPay Server is an open-source self-hosted software stack, not a company. The community and contributors behind BTCPay Server have no control over who uses the software or how they use it.
+If you paid an invoice to a merchant and didn't receive the product/service you paid for, please contact the merchant directly to see what happened.
+
+If the problem arises because of a bug, *the merchant* (not you) should [open an issue on GitHub](https://github.com/btcpayserver/btcpayserver/issues).
```

Comment:
No, anybody can open an issue, I'd remove this part.

Reply:
> No, anybody can open an issue, I'd remove this part.

Got it ! 
Was thinking specifically of a merchant having more informations to provide in case of a bug, where a user wouldn't have much info to work on.

```suggestion

```